packetbeat/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Beats dockerfile used for testing
+FROM ruflin/beat-tester
```

Comment:
I updated the dockerfiles and added a comment. I should now also be more obvious why I added the command at the end. Like this it can build based on the same image from libbeat.